rllib/core/testing/utils.py

```diff
@@ -109,13 +111,20 @@ def get_learner(
         A learner.
 
     """
-
+    # Get our testing (BC) Learner class (given the framework).
     _cls = get_learner_class(framework)
+    # Get our RLModule spec to use.
     spec = get_module_spec(framework=framework, env=env, is_multi_agent=is_multi_agent)
-    # adding learning rate as a configurable parameter to avoid hardcoding it
+    # Adding learning rate as a configurable parameter to avoid hardcoding it
     # and information leakage across tests that rely on knowing the LR value
     # that is used in the learner.
-    return _cls(module_spec=spec, optimizer_config={"lr": learning_rate})
+    learner = _cls(
+        module_spec=spec,
+        learner_group_scaling_config=LearnerGroupScalingConfig(),
```

Comment:
Did we need scaling config? It will get created in the constructor of Learner with default parameters if not specified as far as I remember.